<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagement</class>
 <widget class="QDialog" name="UserManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>393</height>
   </rect>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QPushButton" name="bNewUser">
        <property name="text">
         <string>New User</string>
        </property>
        <property name="icon">
         <iconset resource="reghaabat.qrc">
          <normaloff>:/images/add_user.png</normaloff>:/images/add_user.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bEditUser">
        <property name="text">
         <string>Edit User</string>
        </property>
        <property name="icon">
         <iconset resource="reghaabat.qrc">
          <normaloff>:/images/edit_user.png</normaloff>:/images/edit_user.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bMatchList">
        <property name="text">
         <string>Score List</string>
        </property>
        <property name="icon">
         <iconset resource="reghaabat.qrc">
          <normaloff>:/images/scores.png</normaloff>:/images/scores.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cGenderGroup">
        <property name="text">
         <string>Gender Group</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tUsers"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="reghaabat.qrc"/>
 </resources>
 <connections/>
</ui>
